src/store/chat/bus.ts

```diff
@@ -56,6 +57,8 @@ export function createChatConnection(userId, chatAccessToken) {
       emit({ type: Events.OtherUserJoinedChannel, payload: { channelId, user } });
     const onOtherUserLeftChannel = (channelId, user) =>
       emit({ type: Events.OtherUserLeftChannel, payload: { channelId, user } });
+    const receiveLiveRoomEvent = (roomId, message) =>
+      emit({ type: Events.LiveTimelineEventReceived, payload: { roomId, message } });
```

Comment:
Why did we switch to `Timeline` instead of `Room` for the event name here? Is there a conceptual difference at this boundary?